pyproj/database.pyx

```diff
@@ -483,3 +488,105 @@ def get_database_metadata(str key not None):
         return metadata
     finally:
         pyproj_context_destroy(context)
+
+
+def query_geodetic_crs_from_datum(
+    str crs_auth_name,
+    str datum_auth_name not None,
+    str datum_code not None,
+    pj_type=None
+    ):
+    """
+    .. versionadded:: 3.7.0
+
+    Return GeodeticCRS that use the specified datum
+
+    See: :c:func:`proj_query_geodetic_crs_from_datum`
+
+    Parameters
+    ----------
+    crs_auth_name: str | None
+        The authority name to filter by (e.g. EPSG, ESRI). None is all.
+    datum_auth_name: str
+        The authority of the datum
+    datum_code: str
+        Datum code
+    pj_type: pyproj.enums.PJType | None, optional
+        The type of object to get the CRSs. Can be PJType.GEOCENTRIC_CRS,
+        PJType.GEOGRAPHIC_3D_CRS, PJType.GEOGRAPHIC_2D_CRS or None for all.
+
+    Returns
+    -------
+    list[_CRS]
+    """
+
+    cdef const char* c_crs_type = NULL
+    cdef bytes b_crs_type
+    cdef str crs_type = ""
+    if pj_type is None:
+        pass
+    elif pj_type == PJType.GEOCENTRIC_CRS:
```

Comment:
```suggestion
    elif pj_type is PJType.GEOCENTRIC_CRS:
```

Reply:
done. Is there any difference?

Reply:
It is a minor difference that cython uses to be slightly more performant.